app/templates/views/text-not-received.html

```diff
@@ -3,16 +3,21 @@
 {% from "components/form.html" import form_wrapper %}
 
 {% block per_page_title %}
-Check your mobile number
+If you do not receive a security code
 {% endblock %}
 
 {% block maincolumn_content %}
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="heading-large">Check your mobile number</h1>
+    <h1 class="heading-large">If you do not receive a security code</h1>
 
-    <p class="govuk-body">Check your mobile phone number is correct and then resend the security code.</p>
+    <ol class="govuk-list govuk-list--number">
+      <li>Check that you can receive text messages from other numbers.</li>
+      <li>Switch your phone off and then back on again.</li>
+      <li>Check the mobile phone number you entered is correct.</li>
+      <li>If you still do not receive a security code, we can send you a new one.</li>
+    </ol>    
```

Comment:
Like this @klssmith ?

```suggestion
    <p class="govuk-body">
      <ol class="govuk-list govuk-list--number">
         <li>Check that you can receive text messages from other numbers.</li>
         <li>Switch your phone off and then back on again.</li>
         <li>Check the mobile phone number you entered is correct.</li>
         <li>If you still do not receive a security code, we can send you a new one.</li>
       </ol>
    </p>       
```

Reply:
Sorry, ignore what I said about adding a paragraph actually - we have lots of places where lists are outside paragraphs, so it's fine as it is.

You just need to change [this line](https://github.com/alphagov/notifications-admin/blob/7f3cdce1a6655de7ec858c3b2541aa267371aab3/tests/app/main/views/test_code_not_received.py#L66) to `message = page.select("main ol")[0].text` and the message on the line above will need updating too to match the changes you made.